### ISSUE TITLE
Failure with Gradle 4.0: "You cannot add child specs at execution time"

### ISSUE THREAD

**Issue by joschi** [melix/jmh-gradle-plugin#95]
The JMH plugin seems to fail with Gradle 4.0 with the error message:
```
You cannot add child specs at execution time. Consider configuring this task during configuration time or using a separate task to do the configuration.
```

Running the plugin with Gradle 3.5 works.

**build.gradle**
```
plugins {
    id "me.champeau.gradle.jmh" version "0.3.1"
}

repositories {
    jcenter()
}
```

**Gradle version**
```
$ ./gradlew -v

------------------------------------------------------------
Gradle 4.0
------------------------------------------------------------

Build time:   2017-06-14 15:11:08 UTC
Revision:     316546a5fcb4e2dfe1d6aa0b73a4e09e8cecb5a5

Groovy:       2.4.11
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          1.8.0_131 (Oracle Corporation 25.131-b11)
OS:           Mac OS X 10.12.5 x86_64
```

**Output of `./gradlew jmh`**
```
$ ./gradlew jmh

> Configure project :
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jmhJar'.
> You cannot add child specs at execution time. Consider configuring this task during configuration time or using a separate task to do the configuration.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED in 0s
2 actionable tasks: 1 executed, 1 up-to-date
```

**Comment by melix** [comment]
Thanks for reporting this. It's already fixed, see 7a6423893350d8136e75476fc42b01ea6c48b7d5

**Comment by joschi** [comment]
Awesome, thanks!

Sorry for the noise.
